### PR TITLE
[m0] Mark test cases applicable to M0

### PR DESCRIPTION
**Pull request by yaqiangz**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
Since the newly added topology M0 is similar to T0, add mark to let nightly test case running on T0 topology run on M0 topology.

#### How did you do it?
Add mark.

#### How did you verify/test it?
Run and check nightly test.

### CODE REVIEW

File: tests/pfcwd/test_pfcwd_warm_reboot.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
           }
 
 pytestmark = [pytest.mark.disable_loganalyzer,
-              pytest.mark.topology('t0')
+              pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/pfcwd/test_pfcwd_warm_reboot.py:36]: remove

File: tests/pfc_asym/test_pfc_asym.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import pytest
 
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/pfc_asym/test_pfc_asym.py:5]: remove

File: tests/platform_tests/test_advanced_reboot.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 pytestmark = [
     pytest.mark.disable_loganalyzer,
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/platform_tests/test_advanced_reboot.py:13]: remove

File: tests/platform_tests/test_cont_warm_reboot.py [hunk 4]
```diff
@@ -21,7 +21,7 @@
 
 pytestmark = [
     pytest.mark.disable_loganalyzer,
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/platform_tests/test_cont_warm_reboot.py:24]: remove

File: tests/platform_tests/test_service_warm_restart.py [hunk 5]
```diff
@@ -7,7 +7,7 @@
 
 pytestmark = [
     pytest.mark.disable_loganalyzer,
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/platform_tests/test_service_warm_restart.py:10]: remove

File: tests/qos/test_pfc_counters.py [hunk 6]
```diff
@@ -16,7 +16,7 @@
 """
 
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/qos/test_pfc_counters.py:19]: remove

File: tests/qos/test_pfc_pause.py [hunk 7]
```diff
@@ -16,7 +16,7 @@
 from tests.common.helpers.pfc_storm import PFCStorm
 
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/qos/test_pfc_pause.py:19]: remove

File: tests/qos/test_tunnel_qos_remap.py [hunk 8]
```diff
@@ -11,7 +11,7 @@
 from ptf.testutils import simple_tcp_packet, simple_ipv4ip_packet
 
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/qos/test_tunnel_qos_remap.py:14]: remove

File: tests/span/test_port_mirroring.py [hunk 9]
```diff
@@ -8,7 +8,7 @@
 from span_helpers import send_and_verify_mirrored_packet
 
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/span/test_port_mirroring.py:11]: remove

File: tests/sub_port_interfaces/test_show_subinterface.py [hunk 10]
```diff
@@ -7,7 +7,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology("t0", "t1")
+    pytest.mark.topology("t0", "t1", 'm0')
```
Comment (Blueve) [inline tests/sub_port_interfaces/test_show_subinterface.py:10]: remove

File: tests/sub_port_interfaces/test_sub_port_interfaces.py [hunk 11]
```diff
@@ -17,7 +17,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology('t0', 't1')
+    pytest.mark.topology('t0', 't1', 'm0')
```
Comment (Blueve) [inline tests/sub_port_interfaces/test_sub_port_interfaces.py:20]: remove

File: tests/sub_port_interfaces/test_sub_port_l2_forwarding.py [hunk 12]
```diff
@@ -13,7 +13,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology("t0")
+    pytest.mark.topology("t0", "m0")
```
Comment (Blueve) [inline tests/sub_port_interfaces/test_sub_port_l2_forwarding.py:16]: remove

File: tests/vrf/test_vrf.py [hunk 13]
```diff
@@ -36,7 +36,7 @@
 """
 
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/vrf/test_vrf.py:39]: remove

File: tests/vrf/test_vrf_attr.py [hunk 14]
```diff
@@ -16,7 +16,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/vrf/test_vrf_attr.py:19]: remove

File: tests/vxlan/test_vnet_route_leak.py [hunk 15]
```diff
@@ -12,7 +12,7 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology("t0"),
+    pytest.mark.topology("t0", "m0"),
```
Comment (Blueve) [inline tests/vxlan/test_vnet_route_leak.py:15]: remove

File: tests/vxlan/test_vnet_vxlan.py [hunk 16]
```diff
@@ -22,7 +22,7 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology("t0"),
+    pytest.mark.topology("t0", "m0"),
```
Comment (Blueve) [inline tests/vxlan/test_vnet_vxlan.py:25]: remove

File: tests/vxlan/test_vxlan_decap.py [hunk 17]
```diff
@@ -16,7 +16,7 @@
 from tests.ptf_runner import ptf_runner
 from tests.common.dualtor.mux_simulator_control import mux_server_url, toggle_all_simulator_ports_to_rand_selected_tor_m
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/vxlan/test_vxlan_decap.py:19]: remove

File: tests/arp/test_arp_dualtor.py [hunk 18]
```diff
@@ -12,7 +12,7 @@
 from tests.common.fixtures.ptfhost_utils import change_mac_addresses
 
 pytestmark = [
-    pytest.mark.topology('t0', 'dualtor')
+    pytest.mark.topology('t0', 'dualtor', 'm0')
```
Comment (Blueve) [inline tests/arp/test_arp_dualtor.py:15]: remove

File: tests/bgp/test_bgp_slb.py [hunk 19]
```diff
@@ -8,7 +8,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology("t0"),
+    pytest.mark.topology("t0", "m0"),
```
Comment (Blueve) [inline tests/bgp/test_bgp_slb.py:11]: remove

File: tests/arp/test_wr_arp.py [hunk 20]
```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/arp/test_wr_arp.py:16]: remove

File: tests/dualtor/test_ipinip.py [hunk 21]
```diff
@@ -28,7 +28,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology("t0")
+    pytest.mark.topology("t0", "m0")
```
Comment (Blueve) [inline tests/dualtor/test_ipinip.py:31]: remove

File: tests/dualtor/test_orch_stress.py [hunk 22]
```diff
@@ -29,7 +29,7 @@
 from tests.common.dualtor.dual_tor_mock import *
 
 pytestmark = [
-    pytest.mark.topology("t0")
+    pytest.mark.topology("t0", "m0")
```
Comment (Blueve) [inline tests/dualtor/test_orch_stress.py:32]: remove

File: tests/dualtor/test_orchagent_active_tor_downstream.py [hunk 23]
```diff
@@ -22,7 +22,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/dualtor/test_orchagent_active_tor_downstream.py:25]: remove

File: tests/dualtor/test_orchagent_mac_move.py [hunk 24]
```diff
@@ -17,7 +17,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/dualtor/test_orchagent_mac_move.py:20]: remove

File: tests/dualtor/test_orchagent_standby_tor_downstream.py [hunk 25]
```diff
@@ -26,7 +26,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/dualtor/test_orchagent_standby_tor_downstream.py:29]: remove

File: tests/dualtor/test_server_failure.py [hunk 26]
```diff
@@ -6,7 +6,7 @@
 from tests.common.utilities import wait_until
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/dualtor/test_server_failure.py:9]: remove

File: tests/dualtor/test_standby_tor_upstream_mux_toggle.py [hunk 27]
```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__file__)
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/dualtor/test_standby_tor_upstream_mux_toggle.py:16]: remove

File: tests/dualtor/test_toggle_mux.py [hunk 28]
```diff
@@ -10,7 +10,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology("t0")
+    pytest.mark.topology("t0", "m0")
```
Comment (Blueve) [inline tests/dualtor/test_toggle_mux.py:13]: remove

File: tests/dualtor/test_tor_ecn.py [hunk 29]
```diff
@@ -31,7 +31,7 @@
 from tests.common.dualtor.tunnel_traffic_utils import derive_queue_id_from_dscp
 
 pytestmark = [
-    pytest.mark.topology("t0")
+    pytest.mark.topology("t0", "m0")
```
Comment (Blueve) [inline tests/dualtor/test_tor_ecn.py:34]: remove

File: tests/ecmp/inner_hashing/test_inner_hashing.py [hunk 30]
```diff
@@ -16,7 +16,7 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/ecmp/inner_hashing/test_inner_hashing.py:19]: remove

File: tests/ecmp/inner_hashing/test_inner_hashing_lag.py [hunk 31]
```diff
@@ -15,7 +15,7 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/ecmp/inner_hashing/test_inner_hashing_lag.py:18]: remove

File: tests/ecmp/inner_hashing/test_wr_inner_hashing.py [hunk 32]
```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/ecmp/inner_hashing/test_wr_inner_hashing.py:16]: remove

File: tests/ecmp/inner_hashing/test_wr_inner_hashing_lag.py [hunk 33]
```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/ecmp/inner_hashing/test_wr_inner_hashing_lag.py:16]: remove

File: tests/ecmp/test_fgnhg.py [hunk 34]
```diff
@@ -30,7 +30,7 @@
 DUT_VXLAN_PORT_JSON_FILE = '/tmp/vxlan.switch.json'
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/ecmp/test_fgnhg.py:33]: remove

File: tests/everflow/test_everflow_ipv6.py [hunk 35]
```diff
@@ -9,7 +9,7 @@
 from everflow_test_utilities import setup_info  # noqa: F401, E501 lgtm[py/unused-import] pylint: disable=import-error
 
 pytestmark = [
-    pytest.mark.topology("t0","t1")
+    pytest.mark.topology("t0", "t1", "m0")
```
Comment (Blueve) [inline tests/everflow/test_everflow_ipv6.py:12]: remove

File: tests/everflow/test_everflow_testbed.py [hunk 36]
```diff
@@ -15,7 +15,7 @@
 from tests.common.fixtures.ptfhost_utils import copy_arp_responder_py # noqa: F401, E501 lgtm[py/unused-import] pylint: disable=import-error
 
 pytestmark = [
-    pytest.mark.topology("t0", "t1")
+    pytest.mark.topology("t0", "t1", "m0")
```
Comment (Blueve) [inline tests/everflow/test_everflow_testbed.py:18]: remove

File: tests/nat/test_static_nat.py [hunk 37]
```diff
@@ -43,7 +43,7 @@
 
 
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/nat/test_static_nat.py:46]: remove

File: tests/nat/test_dynamic_nat.py [hunk 38]
```diff
@@ -40,7 +40,7 @@
 from tests.common.helpers.assertions import pytest_assert
 
 pytestmark = [
-    pytest.mark.topology('t0')
+    pytest.mark.topology('t0', 'm0')
```
Comment (Blueve) [inline tests/nat/test_dynamic_nat.py:43]: remove

File: tests/macsec/test_macsec.py [hunk 39]
```diff
@@ -17,7 +17,7 @@
 
 pytestmark = [
     pytest.mark.macsec_required,
-    pytest.mark.topology("t0", "t2"),
+    pytest.mark.topology("t0", "t2", 'm0'),
```
Comment (Blueve) [inline tests/macsec/test_macsec.py:20]: remove

File: tests/restapi/test_restapi.py [hunk 40]
```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 pytestmark = [
-    pytest.mark.topology('t0'),
+    pytest.mark.topology('t0', 'm0'),
```
Comment (Blueve) [inline tests/restapi/test_restapi.py:16]: remove

**Review by Blueve — CHANGES_REQUESTED** [review]

File: tests/pfcwd/test_pfcwd_warm_reboot.py (hunk 1 above)
Comment (yaqiangz) [reply tests/pfcwd/test_pfcwd_warm_reboot.py]: Fixed

File: tests/pfc_asym/test_pfc_asym.py (hunk 2 above)
Comment (yaqiangz) [reply tests/pfc_asym/test_pfc_asym.py]: Fixed

File: tests/platform_tests/test_cont_warm_reboot.py (hunk 4 above)
Comment (yaqiangz) [reply tests/platform_tests/test_cont_warm_reboot.py]: Fixed

File: tests/platform_tests/test_service_warm_restart.py (hunk 5 above)
Comment (yaqiangz) [reply tests/platform_tests/test_service_warm_restart.py]: Fixed

File: tests/everflow/test_everflow_testbed.py (hunk 36 above)
Comment (yaqiangz) [reply tests/everflow/test_everflow_testbed.py]: Fixed

File: tests/restapi/test_restapi.py (hunk 40 above)
Comment (yaqiangz) [reply tests/restapi/test_restapi.py]: Fixed

File: tests/macsec/test_macsec.py (hunk 39 above)
Comment (yaqiangz) [reply tests/macsec/test_macsec.py]: Fixed

File: tests/nat/test_dynamic_nat.py (hunk 38 above)
Comment (yaqiangz) [reply tests/nat/test_dynamic_nat.py]: Fixed

File: tests/nat/test_static_nat.py (hunk 37 above)
Comment (yaqiangz) [reply tests/nat/test_static_nat.py]: Fixed

File: tests/qos/test_pfc_counters.py (hunk 6 above)
Comment (yaqiangz) [reply tests/qos/test_pfc_counters.py]: Fixed

File: tests/dualtor/test_orchagent_standby_tor_downstream.py (hunk 25 above)
Comment (yaqiangz) [reply tests/dualtor/test_orchagent_standby_tor_downstream.py]: Fixed

File: tests/dualtor/test_orchagent_mac_move.py (hunk 24 above)
Comment (yaqiangz) [reply tests/dualtor/test_orchagent_mac_move.py]: Fixed

File: tests/dualtor/test_orchagent_active_tor_downstream.py (hunk 23 above)
Comment (yaqiangz) [reply tests/dualtor/test_orchagent_active_tor_downstream.py]: Fixed

File: tests/dualtor/test_orch_stress.py (hunk 22 above)
Comment (yaqiangz) [reply tests/dualtor/test_orch_stress.py]: Fixed

File: tests/dualtor/test_ipinip.py (hunk 21 above)
Comment (yaqiangz) [reply tests/dualtor/test_ipinip.py]: Fixed

File: tests/arp/test_wr_arp.py (hunk 20 above)
Comment (yaqiangz) [reply tests/arp/test_wr_arp.py]: Fixed

File: tests/bgp/test_bgp_slb.py (hunk 19 above)
Comment (yaqiangz) [reply tests/bgp/test_bgp_slb.py]: Fixed

File: tests/arp/test_arp_dualtor.py (hunk 18 above)
Comment (yaqiangz) [reply tests/arp/test_arp_dualtor.py]: Fixed

File: tests/vxlan/test_vxlan_decap.py (hunk 17 above)
Comment (yaqiangz) [reply tests/vxlan/test_vxlan_decap.py]: Fixed

File: tests/vxlan/test_vnet_vxlan.py (hunk 16 above)
Comment (yaqiangz) [reply tests/vxlan/test_vnet_vxlan.py]: Fixed

File: tests/vxlan/test_vnet_route_leak.py (hunk 15 above)
Comment (yaqiangz) [reply tests/vxlan/test_vnet_route_leak.py]: Fixed

File: tests/vrf/test_vrf_attr.py (hunk 14 above)
Comment (yaqiangz) [reply tests/vrf/test_vrf_attr.py]: Fixed

File: tests/vrf/test_vrf.py (hunk 13 above)
Comment (yaqiangz) [reply tests/vrf/test_vrf.py]: Fixed

File: tests/sub_port_interfaces/test_sub_port_l2_forwarding.py (hunk 12 above)
Comment (yaqiangz) [reply tests/sub_port_interfaces/test_sub_port_l2_forwarding.py]: Fixed

File: tests/sub_port_interfaces/test_sub_port_interfaces.py (hunk 11 above)
Comment (yaqiangz) [reply tests/sub_port_interfaces/test_sub_port_interfaces.py]: Fixed

File: tests/sub_port_interfaces/test_show_subinterface.py (hunk 10 above)
Comment (yaqiangz) [reply tests/sub_port_interfaces/test_show_subinterface.py]: Fixed

File: tests/span/test_port_mirroring.py (hunk 9 above)
Comment (yaqiangz) [reply tests/span/test_port_mirroring.py]: Fixed

File: tests/qos/test_tunnel_qos_remap.py (hunk 8 above)
Comment (yaqiangz) [reply tests/qos/test_tunnel_qos_remap.py]: Fixed

File: tests/qos/test_pfc_pause.py (hunk 7 above)
Comment (yaqiangz) [reply tests/qos/test_pfc_pause.py]: Fixed

File: tests/platform_tests/test_advanced_reboot.py (hunk 3 above)
Comment (yaqiangz) [reply tests/platform_tests/test_advanced_reboot.py]: Fixed

File: tests/dualtor/test_server_failure.py (hunk 26 above)
Comment (yaqiangz) [reply tests/dualtor/test_server_failure.py]: Fixed

File: tests/dualtor/test_standby_tor_upstream_mux_toggle.py (hunk 27 above)
Comment (yaqiangz) [reply tests/dualtor/test_standby_tor_upstream_mux_toggle.py]: Fixed

File: tests/dualtor/test_toggle_mux.py (hunk 28 above)
Comment (yaqiangz) [reply tests/dualtor/test_toggle_mux.py]: Fixed

File: tests/dualtor/test_tor_ecn.py (hunk 29 above)
Comment (yaqiangz) [reply tests/dualtor/test_tor_ecn.py]: Fixed

File: tests/ecmp/inner_hashing/test_inner_hashing.py (hunk 30 above)
Comment (yaqiangz) [reply tests/ecmp/inner_hashing/test_inner_hashing.py]: Fixed

File: tests/everflow/test_everflow_ipv6.py (hunk 35 above)
Comment (yaqiangz) [reply tests/everflow/test_everflow_ipv6.py]: Fixed

File: tests/ecmp/test_fgnhg.py (hunk 34 above)
Comment (yaqiangz) [reply tests/ecmp/test_fgnhg.py]: Fixed

File: tests/ecmp/inner_hashing/test_wr_inner_hashing_lag.py (hunk 33 above)
Comment (yaqiangz) [reply tests/ecmp/inner_hashing/test_wr_inner_hashing_lag.py]: Fixed

File: tests/ecmp/inner_hashing/test_wr_inner_hashing.py (hunk 32 above)
Comment (yaqiangz) [reply tests/ecmp/inner_hashing/test_wr_inner_hashing.py]: Fixed

File: tests/ecmp/inner_hashing/test_inner_hashing_lag.py (hunk 31 above)
Comment (yaqiangz) [reply tests/ecmp/inner_hashing/test_inner_hashing_lag.py]: Fixed

File: tests/dualtor/test_orchagent_mac_move.py (hunk 24 above)
Comment (yaqiangz) [reply tests/dualtor/test_orchagent_mac_move.py]: Fixed

**Review by Blueve — APPROVED** [review]

**Pull request merged**